zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/Interpreter.java

```diff
@@ -249,15 +249,22 @@ public Interpreter getInterpreterInTheSameSessionByClassName(String className) {
     private String name;
     //@SerializedName("interpreterClassName")
     private String className;
+    private boolean defaultInterpreter;
```

Comment:
The original way to order interpreters is matching from `zeppelin.interpreters`. I, However, think we don't build with some of interpreters of same interpreter group. For example, We don't build Zeppelin with `spark.sql` excluding `spark.spark`. Then, we don't need to have full list of interpreters, and it would be ok to have a interpreter group orders and the information of which interpreter is used by default. `defaultInterpreter` indicates if this interpreter is default interpreter in a same interpreter group or not. If it sets, its interpreter will be located in front of the list of interpreter settings and interpreter gourp info.